skimage/filters/_window.py

```diff
@@ -82,37 +80,48 @@ def window(window_type, size, ndim=2, warp_kwargs=None):
     Return a Hann window with shape (512, 512):
 
     >>> from skimage.filters import window
-    >>> w = window('hann', 512)
+    >>> w = window('hann', (512, 512))
 
     Return a Kaiser window with beta parameter of 16 and shape (256, 256, 256):
 
-    >>> w = window(16, 256, ndim=3)
+    >>> w = window(16, (256, 256, 256))
 
     Return a Tukey window with an alpha parameter of 0.8 and shape (100, 100):
 
-    >>> w = window(('tukey', 0.8), 100)
+    >>> w = window(('tukey', 0.8), (100, 100))
```

Comment:
Might as well change one of these examples to be anisotropic?

Reply:
I have to merge in from master as I am hitting some build error on the travis CI, and will make this change at that time.

Reply:
done!